### ISSUE TITLE
Fastclick/CodeMirror incompatibility

### ISSUE THREAD

**Issue by horner** [meteor/meteor#5922]
@stubailo:

See https://github.com/codemirror/CodeMirror/issues/3751

When fastclick is on the page with CodeMirror, clicks for focus are ignored. Others have documented the problem in their projects: TryGhost/Ghost#5717

There is no good work-around.  The documented fastclick method of using class="needsclick" won't work since CodeMirror makes all of it's UI dynamically.

Made a PR on fastclick: here: https://github.com/ftlabs/fastclick/pull/448

**Comment by stubailo** [comment]
Seems like a simple workaround would be to only include Fastclick in Cordova builds, which this package does for you: https://github.com/meteor/meteor/tree/devel/packages/mobile-experience

I'm not sure that there is anything we can do here, is your goal to keep this issue open until the PR to Fastclick is accepted?

**Comment by horner** [comment]
I agree that Fastclick should fix and then meteor should do a pull. 

I included the mobile experience package... However I wanted CodeMirror to work on iOS. Android works fine since Fastclick disables when the viewport is not set to scale, but iOS does not and there is no good way to disable. I ended up copying the Fastclick code from the meteor project into my packages and hacking it to disable it. 

Trying to save others the 3 days of trouble shooting it took me to get CodeMirror to work.
